objects/tag.c

```diff
@@ -139,7 +139,6 @@ tag_view(lua_State *L, int udx, bool view)
     {
         tag->selected = view;
         banning_need_update();
-        ewmh_update_net_current_desktop();
```

Comment:
This line is still needed, I think, for the case when no client is currently focused and you are switching between tags (and the tag you switch to also doesn't have any clients).